lib/packageModules.js

```diff
@@ -113,7 +155,7 @@ function setServiceArtifactPath(artifactPath) {
   _.set(this.serverless, 'service.package.artifact', artifactPath);
 }
 
-function isIndividialPackaging() {
+function isIndividualPackaging() {
```

Comment:
👍🏼 😄

Reply:
@j0k3r It's all greens, I wonder if we need more coverage?

Reply:
Well, the whole new zip generation seems to not be tested at all 🤔
https://coveralls.io/builds/43972011/source?filename=lib%2FpackageModules.js#L45

Reply:
https://coveralls.io/builds/43972011/source?filename=lib%2FpackageModules.js#L199 calls to `isIndividualPackaging` should be throwing but it's not 🤔🤔🤔

Reply:
I don't have capacity to update this any further. If you are looking for test coverage, check serverless' code where it was copied from. Maybe there are tests that can be copied as well.

The current implementation doesn't work, this one does, and we're already using this fork for ourselves. Hopefully someone will carry the torch to satisfy the reviewers and get this merged.

Reply:
@russell-dot-js not a problem at all, please consider this thread resolved.